Dockerfile

```diff
@@ -1,11 +1,15 @@
 ##### BASE IMAGE #####
-FROM python:3.7-slim-stretch
+# FROM python:3.7-slim-stretch
+FROM elixircloud/foca:20220524-py3.7
```

Comment:
Might as well try to use a newer version of Python if it doesn't break in too many places. Try `3.10` ideally.

Reply:
Yes, I have tried all versions and updated the image to '3.10'.